api-server/Web/Controllers/GitHubController.cs

```diff
@@ -368,21 +368,17 @@ public async Task<ActionResult> getRepository()
 
         var userClient = GetNewClient(access_token);
 
-        // Get organizations for the current user
-        var organizations = await userClient.Organization.GetAllForCurrent(); // organization.Login gibi data çekebiliyoruz
-        var organizationLogins = organizations.Select(org => org.Login).ToArray();
-
+        var repos = userClient.Repository.GetAllForCurrent().Result.Select(repo => repo.Id).ToList();
```

Comment:
This requires that the organization has authorized the HubReview OAuth app (not the GitHub App):

https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/about-oauth-app-access-restrictions

Reply:
Not saying this is an issue, it's just important to know this.

Reply:
Eğer uzun vadede projeyi patlatma olasılığı varsa engellemeye çalışayım. Tam olarak sıkıntının ne olduğunu anlayamadım.